intelmq/bots/experts/securitytxt/expert.py

```diff
@@ -0,0 +1,104 @@
+from typing import Optional
+
+import requests
+from securitytxt import SecurityTXT
+
+from intelmq.lib.bot import ExpertBot
+
+
+class SecurityTXTExpertBot(ExpertBot):
+    """
+    A bot for retrieving contact details from a security.txt
+    """
+    """
+    url_field: The field where to find the url which should be searched
+    contact_field: Field in which to place the found contact details
+
+    only_email_address: whether to select only email addresses as contact detail (no web urls)
+    overwrite: whether to override existing data
+    check_expired / check_canonical: whether to perform checks on expiry date / canonical urls.
+    """
+    url_field: str = "source.reverse_dns"
+    contact_field: str = "source.abuse_contact"
+
+    only_email_address: bool = True
+    overwrite: bool = True
+    check_expired: bool = False
+    check_canonical: bool = False
+
+    def init(self):
+        if not self.url_field or not self.contact_field:
+            raise AttributeError("Not all required fields are set.")
```

Comment:
```suggestion
    def init(self):
        if not SecurityTXT:
            raise MissingDependencyError('wellknown-securitytxt')
```
The parameters are always given as there are default values.
Adding an understandable error message if the dependency is not fulfilled.